test/test_browser.py

```diff
@@ -2999,6 +2999,10 @@ def in_html(expected):
 
         in_html('200')
 
+  @requires_graphics_hardware
+  def test_glfw3_default_hints(self):
+      self.btest_exit('test_glfw3_default_hints.c', args=['-sUSE_GLFW=3', '-lglfw', '-lGL'])
```

Comment:
We use 2-space indentation for python code

Reply:
fixed